resources/user-guides/getting-started-with-adobe-experience-platform/sample-xdm-implementation.md

```diff
@@ -38,21 +38,23 @@ Set the `LAUNCH_ENVIRONMENT_FILE_ID` to the copied Environment File ID in the `M
    * Select **Create schema**, followed by **XDM ExperienceEvent**
    * Set the name for this schema as "Product Reviews".
    * Select the **Add** button (+) to add field groups
+     
      * Select **Environment Details** from the "Use existing field groups" section then select **Add field group**.
    * Select the **Add** button (+) to add field groups
+     
      * Select **Create new field group** and set the name as "Product review", then select **Add field group**.
-   * In the Schema structure, select the "Product review" field group and select the plus icon (+) next to the schema name "Product Reviews".
+* In the Schema structure, select the "Product review" field group and select the plus icon (+) next to the schema name "Product Reviews".
```

Comment:
Did you intend to remove the leading space here? From GH's viewer it's hard to tell if this will cause indentation issues when displaying the list in GitBook.